sample_size/metrics.py

```diff
@@ -31,6 +34,21 @@ def check_positive(number: float, name: str) -> float:
         else:
             return number
 
+    @abstractmethod
+    def generate_p_value(self, true_null: bool, sample_size: int, variants: int) -> List[float]:
+        """
+        This method simulates registered metric's p-value(s). The output will later be applied to BH procedure
+
+        Parameters:
+            true_null: whether the null hypothesis is true
+            sample_size: sample size used for simulations
+            variants: number of test variants, including control
+
+        Returns:
+            p-value(s): the p-values generated. This is a float or a list with length variants-1
```

Comment:
nit: The return is set to a list. Would it ever be a float?

Reply:
It can be a list with one element but is always a list. I have changed this docstring to avoid confusion